### ISSUE TITLE
Add a flag value selector to the redux bindings

### ISSUE THREAD

**Issue by zsgalusz** [garbles/flag#28]
We currently use flags with redux and redux-saga. A big shortcoming we found with the setup is that while flags works great inside the redux components, we have no good way to check a flag's state from the redux middleware. 

It would be very helpful to have a selector to check the value of a flag in the store - while we can brew our own, we need to duplicate all the logic used for calculated flags in the react component, and hope it will never diverge.

**Comment by garbles** [comment]
Good idea. Should be easy to add. This is what I'm thinking

```ts
import createReduxBindings from "flag/redux";

const { getFlagsSelector } = createReduxBindings(/* ... */);

// ...

const flags = getFlagsSelector(state);
```

**Comment by zsgalusz** [comment]
`flags` there will be the computed flags, right? If so, sounds good!

**Comment by garbles** [comment]
@zsgalusz eh so I published a prerelease with this. 

https://github.com/garbles/flag/releases/tag/v4.1.0-0

If you can test this out and let me know whether it works, I'll cut `4.1.0`. Thanks!

**Comment by zsgalusz** [comment]
Looks good! Thank you!

**Comment by garbles** [comment]
Published `4.1.0`. Thanks again for the suggestion!
